config.json

```diff
@@ -153,6 +154,11 @@
       "topics": []
     },
     {
+      "slug": "meetup",
+      "difficulty": 1,
+      "topics": ["Dates"]
```

Comment:
I really like that you added a topic. Especially because it is in the [standard topics list](https://github.com/exercism/x-common/blob/master/TOPICS.txt#L24).